webpack.config.js

```diff
@@ -68,6 +68,9 @@ module.exports = {
     output: {
         // Specify the destination of all build products.
         path: distDir,
+        // base path for build products when referenced from production
+        // (see https://webpack.js.org/guides/code-splitting/)
```

Comment:
That link doesn't actually mention this parameter :P Use this one: https://webpack.js.org/guides/public-path/ (or no link is also ok)

Reply:
done